src/univariates.jl

```diff
@@ -37,6 +37,10 @@ end
 @compat support{D<:ContinuousUnivariateDistribution}(d::Union{D,Type{D}}) = RealInterval(minimum(d), maximum(d))
 @compat support{D<:DiscreteUnivariateDistribution}(d::Union{D,Type{D}}) = round(Int, minimum(d)):round(Int, maximum(d))
 
+# Type used for dispatch on finite support
+# T = true or false
+immutable FiniteSupport{T} end
```

Comment:
Yeah, I probably should have waited--sorry about that.  I was impatient to tag a new version, and didn't want to tag multiple versions with small changes in a row.  But I'll wait in the future.

Possibly true, also, that a simple branch would have done the trick.  Not sure I'll have time today, but I'll try that change when I get the chance.